android/src/main/java/com/urbanairship/reactnative/ReactNotificationProvider.java

```diff
@@ -11,17 +11,17 @@
 import com.urbanairship.AirshipConfigOptions;
 import com.urbanairship.push.notifications.AirshipNotificationProvider;
 
```

Comment:
Minor code cleanup, missing nullabilty annotation and making things package protected since public is not needed